### DISCUSSION TITLE
Custom Components' imports not wroking

### DISCUSSION THREAD

**Discussion by obeduri** [gilbarbara/react-joyride#964] [Q&A] [answered]
While using the library I came across a request to modify some UI elements in the tooltip component, I read the docs and found: https://docs.react-joyride.com/custom-components#tooltipcomponent 

I added the sample code to my own just to make it work and start my requirement from there. Somehow I cannot find where the components: TooltipBody, TooltipTitle, TooltipContent, and others are from the example code are.

I searched Joyride's code globally with Command + Shift + F and couldn't find any of them... I think I'm missing something I guess. ˆˆ'' please help

**Comment by gilbarbara** [comment, marked as answer]
Hey @obeduri 

You'd need to create those components yourself. That's the point of customization.

If you want to use the default layout, don't set the `tooltipComponent` prop.

**Reply by obeduri** [reply]
if TooltipBody, TooltipTitle, TooltipContent, etc... do not exist... then might as well use divs for the example so people won't think TooltipBody, TooltipTitle, TooltipContent are components that are in the library ˆˆ''?

Thank you for the help is working amazing :) thanks for the awesome library #uDaBest cheers from Dominican Republic
